library/src/app/components/trade/trade.component.html

```diff
@@ -0,0 +1 @@
+<p>Trade Component</p>
```

Comment:
Localized

Reply:
Removed. You can't route to it for now anyways.